### PR TITLE
Add target and env to modifyBabelOptions

### PR DESCRIPTION
**Pull request by revskill10**

### CODE REVIEW

**Comment by aga5tya** [comment]
@jaredpalmer Could you please review this, would help in my use-case too where I have set of babel plugins that is needed only on the server compilation.

File: packages/razzle/config/createConfig.js [hunk 1]
```diff
@@ -54,6 +54,8 @@ module.exports = (
     babelrc: true,
     cacheDirectory: true,
     presets: [],
+    target,
+    env,
```
Comment (aga5tya) [inline packages/razzle/config/createConfig.js]: We could move this as an additional argument to modifyBabelOptions ?

modifyBabelOptions(mainBabelOptions, { target, env })

Comment (revskill10) [reply packages/razzle/config/createConfig.js]: I moved them to a `buildOptions`.

**Comment by phelma** [comment]
This would be really useful to have seperate babel configs for browser and node, specifically seperate @babel/preset-env browserslist targets.

**Comment by nimaa77** [comment]
Hello and thanks for your contribution we have the same config in #1108 and it's merged.

so I'm going to close this at this moment.

**Pull request closed without merging**